dds/DCPS/transport/framework/TransportInst.cpp

```diff
@@ -85,11 +85,48 @@ OpenDDS::DCPS::TransportInst::formatNameForDump(const char* name)
       << name << ": ";
   return oss.str();
 #else
-  ACE_UNUSED_ARG(name);
-  return "not implemented";
+  OPENDDS_STRING formatted_name;
+  formatted_name.reserve(NAME_INDENT + NAME_WIDTH);
+  for (int i = 0; i < NAME_INDENT; ++i) {
+    formatted_name.append(" ");
+  }
+  formatted_name.append(name);
+  OPENDDS_STRING delim(": ");
+  formatted_name.append(delim);
+  for (int i = 0; i < (NAME_WIDTH - strlen(name) - delim.length()); ++i) {
+    formatted_name.append(" ");
+  }
+  return formatted_name;
 #endif
 }
 
+OPENDDS_STRING
+OpenDDS::DCPS::TransportInst::dump_to_str()
+{
+  OPENDDS_STRING ret;
+  ret += formatNameForDump("transport_type")          += this->transport_type_;
```

Comment:
The second += can just be +.  I suppose they are equivalent but it's not commonly written that way.